litellm/llms/vertex_ai_anthropic.py

```diff
@@ -237,7 +237,7 @@ def completion(
             if vertex_credentials is not None and isinstance(vertex_credentials, str):
                 import google.oauth2.service_account
 
-                json_obj = json.loads(vertex_credentials)
+                json_obj = json.load(open(vertex_credentials))
```

Comment:
hey @hawktang this assumes the vertex_credentials is a filepath 

That's not the intended purpose of this code block. It's a json str of the vertex_credentials 

I believe we can support both flows - can you change this to check if it's a filepath or json string and proceed accordingly

Reply:
example for intended flow - 

https://github.com/BerriAI/litellm/blob/056913fd7049923a106130b02d7c29e7f312beec/litellm/tests/test_amazing_vertex_completion.py#L130

Reply:
Looking further at the example - do you even need to set this? 

If you have `GOOGLE_APPLICATION_CREDENTIALS` in your env, the vertex ai sdk should automatically be handling this?